3.10/installation-windows.md

```diff
@@ -12,6 +12,7 @@ You can install ArangoDB on Windows (64-bit) using different methods:
   - [attended](#installing-using-the-installer) (GUI)
   - [unattended](#unattended-installation-using-the-installer) (command line)
 - Manually, using a [ZIP archive](#installing-using-the-zip-archive)
+- Using a [Docker image](#docker)
```

Comment:
> You can install ArangoDB on Windows (64-bit) using different methods:

This should be reworded as you don't really "install" Docker images ("You can use..."), and since Docker will be recommended way of using ArangoDB on Windows, I would put Docker images at the top of the list (which then also reflects the order of headlines)